examples/server.js

```diff
@@ -10,7 +10,7 @@ config.entry.app.push(
 
 new WebpackDevServer(webpack(config), {
   publicPath: config.output.publicPath,
-  historyApiFallback: true,
```

Comment:
Are you sure that need removing (`historyApiFallback`)?

Reply:
Yeah, I am sure.

Reply:
I checked it. There is no need to removing  (historyApiFallback). Add (historyApiFallback) release the limit to the host. ^_^